Resources/public/js/views/services/plugins/ez-versionsplugin.js

```diff
@@ -136,19 +136,19 @@ YUI.add('ez-versionsplugin', function (Y) {
         },
 
         /**
-         * deleteVersionDraft event handler, opens confirm box to confirm that selected draft
+         * deleteVersion event handler, opens confirm box to confirm that selected versions
          * are going to be deleted
          *
-         * @method _deleteDraftConfirm
+         * @method _deleteVersionConfirm
          * @private
-         * @param {EventFacade} e deleteVersionDraft event facade
+         * @param {EventFacade} e deleteVersion event facade
          */
-        _deleteDraftConfirm: function (e) {
+        _deleteVersionConfirm: function (e) {
```

Comment:
I missed it in the previous review but it should probably be called `_confirmDeleteVersion` instead